test/data.jl

```diff
@@ -71,6 +71,37 @@ module TestData
     sdf6f = view(df6, UInt64[1, 2])
     @test size(sdf6f) == (2,3)
 
+    #test_group("getindex")
+    # ensure indexing column by a single item (by integer/symbol) results in a DataArray
+    v6a = sdf6a[1]
+    @test isa(v6a, DataArray)
```

Comment:
`DataArray` will have to be replaced with `Array{Union{T, Null}}`. This is what makes the tests fail.